assets/common/src/batch_deploy.py

```diff
@@ -194,18 +194,18 @@ def invoke_endpoint_job(ml_client, endpoint, type, args):
         scoring_job = list(ml_client.jobs.list(parent_job_name=job.name))[0]
 
         ml_client.jobs.download(
-            name=scoring_job.name, download_path=".", output_name="score"
+            name=scoring_job.name, download_path=args.batch_job_output_folder, output_name="score"
```

Comment:
Can you share how output folder would look like? 
Is it: 

batch_job_output_folder
| - named-outputs
| - - - - - - - score
| - - - - - - - - - - - prediction.csv
| - model_deployment_details.json

Reply:
Yes it looks like this

Reply:
Could we capture it like this: 

batch_job_output_folder
| - - score
| - - - - - prediction.csv
| - model_deployment_details.json

Reply:
Done